packages/is-shallow-equal/src/index.js

```diff
@@ -13,10 +13,10 @@ export { default as isShallowEqualArrays } from './arrays';
 
 /**
  * Returns true if the two arrays or objects are shallow equal, or false
- * otherwise.
+ * otherwise. Also handles primitive values, just in case.
  *
- * @param {any[]|ComparableObject} a First object or array to compare.
- * @param {any[]|ComparableObject} b Second object or array to compare.
+ * @param {any} a First object or array to compare.
+ * @param {any} b Second object or array to compare.
```

Comment:
not to make this more confusing, but isn't `unknown` possibly more useful here than `any`? whereas `any` satisfies all type constraints, the `unknown` would only widen as we make programmatic checks to assert its type.

Reply:
Can do!